include/exadg/grid/one_sided_cylindrical_manifold.h

```diff
@@ -47,7 +47,7 @@ class OneSidedCylindricalManifold : public dealii::ChartManifold<dim, dim, dim>
                               dealii::Point<dim> const &                                 center_in)
     : alpha(1.0), radius(1.0), cell(cell_in), face(face_in), center(center_in)
   {
-    AssertThrow(face <= 3,
+    AssertThrow(face >= 0 and face <= 3,
```

Comment:
First check is actually unnecessary and will always be true since `face` is `unsigned int`. Also the case for line 366. Do you still want to keep it?

Reply:
Ah, I did not want to make this change. I solved merge conflicts and selected the wrong version.